src/modules/home/components/property/property-card.tsx

```diff
@@ -13,17 +15,48 @@ import 'swiper/css/pagination';
 import './swiper.css';
 
 import { EffectFade, Navigation, Pagination } from 'swiper/modules';
+import { axiosInstance } from '#/configs';
+import { useWishlistedProperties } from '../../api/get-wishlisted-properties';
 
-export function ApartmentCard(apartment: Accommodation) {
-  const handleFavoriteClick = (event: React.MouseEvent) => {
+export function ApartmentCard({ id, name, location, pricePerNight, images }: Accommodation) {
+  const { data } = useWishlistedProperties();
+  const navidate = useNavigate();
+  const [isLiked, setIsLiked] = useState(false);
```

Comment:
Give a more specific name since it is not actually liking the accommodation but adding it to the wishlist. For example, `isInWishlisht` or `addedToWishlist`